Makefile

```diff
@@ -61,7 +61,7 @@ ifeq (, $(shell which controller-gen))
 	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
 	cd $$CONTROLLER_GEN_TMP_DIR ;\
 	go mod init tmp ;\
-	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.5 ;\
+	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1 ;\
```

Comment:
This version or later will be required to update `AdmissionReviewVersions` from annotation. (Please also see https://github.com/kubernetes-sigs/controller-tools/commit/1dc8cc4f84c85b8f90aa58a87cbb768000f3518c)